.github/workflows/actions/manage-doppler-secret/action.yml

```diff
@@ -26,7 +26,7 @@ runs:
     - name: 'Execute Doppler Secret Management script'
       env:
         DOPPLER_TOKEN: ${{ inputs.doppler-token }}
-      if: ${{ inputs.doppler-project == 'pool' || inputs.doppler-project == 'url-shortener' ||  inputs.doppler-project == 'sourcery' || inputs.doppler-project == 'wheregologin' }}
+      if: ${{ inputs.doppler-token != 0 && inputs.doppler-token != '' }}
```

Comment:
It looks like doppler token is required in this action..
It also looks required in the action consuming it

https://github.com/onboardiq/gh-common-workflows/blob/4ac8384f3ee2f7326a896452fca770ab6739d576/.github/workflows/actions/deploy-to-tenant/action.yaml#L64-L67

Perhaps we need to remove a required on the parent action and then only run the action deploy-to-tenant if a doppler-token is provided instead?

Reply:
changed here: [remove requires](https://github.com/onboardiq/gh-common-workflows/pull/23/commits/4f4d2cf1ec118f3fea8eb0d608584f30ce638ed4) and [move if one action up](https://github.com/onboardiq/gh-common-workflows/pull/23/commits/343a704e3ac8a7f96c45b8a05f9c4cc2bcd71afc)